addons/binding/org.openhab.binding.feed.test/src/test/java/org/openhab/binding/feed/test/EmptyTest.java

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
Why is this required?
Aren't the tests working if you don't have at least one Java test along the Groovy tests?

In ESH, we have [test fragments without any Java tests](https://github.com/eclipse/smarthome/tree/master/bundles/core/org.eclipse.smarthome.core.binding.xml.test/src/test), so I would assume you do not need this.